packages/vite/src/node/plugins/completeSystemWrap.ts

```diff
@@ -0,0 +1,22 @@
+import type { Plugin } from '../plugin'
+
+/**
+ * make sure systemjs register wrap to had complete parameters in system format
+ */
+export function completeSystemWrapPlugin(): Plugin {
+  const SystemJSWrapRE = /System.register\(.*\((exports)\)/g
+
+  return {
+    name: 'vite:force-systemjs-wrap-complete',
+
+    renderChunk(code, chunk, opts) {
+      if (opts.format === 'system') {
+        return {
+          code: code.replace(SystemJSWrapRE, (s, s1) =>
+            s.replace(s1, 'exports, module')
+          )
```

Comment:
```suggestion
          code: code.replace(SystemJSWrapRE, (s, s1) =>
            s.replace(s1, 'exports, module')
          ),
          map: null
```
We need `map` to avoid `Sourcemap is likely to be incorrect: a plugin (vite:force-systemjs-wrap-complete) 
was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help` warning.